WcaOnRails/app/views/competitions/_competition_info.html.erb

```diff
@@ -75,6 +75,12 @@
       </dd>
     </dl>
 
+	<dl class="dl-horizontal">
+	  <dt><%= t 'competitions.nav.menu.competitors' %></dt>
+	  <dd><%= @competition.competitors.count %></dd>
+	  </dd>
```

Comment:
What is this `</dd>` closing? Is it an orphan?

Reply:
I think I had the <dd> split into lines, but then moved it to a single one and forgot to remove this.